tests/__init__.py

```diff
@@ -7,3 +7,11 @@ def get_examples_invalid_user_repo() -> typing.List[str]:
         "foo/another_bar",
         "another_foo/another_bar"
     ]
+
+
+def get_wrong_href_content() -> typing.List[str]:
+    return ["/", "a", "baz"]
```

Comment:
The "" case is treated by the unit-tests for the cases where it raises the MissingHrefAttributeError Exception.